### ISSUE TITLE
Doesn't work on Windows 10

### ISSUE THREAD

**Issue by 6XGate** [LinusU/vscode-auto-dark-mode#10]
I've tried toggling this in Settings > Personalization > Colors and VSCode does not respond.

**Comment by GrzegorzWidla** [comment]
For Windows 10 use [VS Code Auto Dark Mode Windows](https://github.com/danielgjackson/vscode-auto-dark-mode-windows) extension but it would be great to have them combined into a single one as I - and probably many others - sync their settings between multiple devices and operating systems.

**Comment by LinusU** [comment]
Would be happy to add support for this, but don't have Windows 10 myself.

@6XGate @GrzegorzWidla could you help me out with trying this?

Could you paste the following into a file and save it as e.g. `dark-mode-listener.vbs`, then run it with `cscript.exe dark-mode-listener.vbs` in the command prompt, and see that it 1) prints the current mode upon launch, and then 2) prints a new line every time that you toggle dark mode on or off.

```vbs
Set WSHShell = CreateObject("WScript.Shell")
Set WMIService = GetObject("winmgmts:\\.\root\default")

function printCurrentMode ()
  On Error Resume Next

  Set value = WSHShell.RegRead("HKEY_CURRENT_USER\\SOFTWARE\\Microsoft\\Windows\\CurrentVersion\\Themes\\Personalize\\AppsUseLightTheme")

  if err.number <> 0 then
    wscript.echo "light"
  else
    if value <> 0 then
      wscript.echo "light"
    else
      wscript.echo "dark"
    end if
  end if
end function

printCurrentMode()

Set colEvents = WMIService.ExecNotificationQuery("SELECT * FROM RegistryKeyChangeEvent WHERE Hive='HKEY_CURRENT_USER' AND KeyPath='SOFTWARE\\Microsoft\\Windows\\CurrentVersion\\Themes\\Personalize\\AppsUseLightTheme'")
Do
  Set latestEvent = colEvents.NextEvent
  printCurrentMode()
Loop
```

If this works it should be easy to integrate! I've never used VisualBasicScript (or WMI) though so it probably won't work out of the box 😄

**Comment by LinusU** [comment]
I think I got it working, with a lot of modifications 😁 

The latest script is here, would be great if you could test that it works and leave a comment on the PR: https://github.com/LinusU/node-dark-mode-listener/pull/4

**Comment by LinusU** [comment]
Fixed in 0.1.7 🎉 

<img width="474" alt="Screenshot 2019-11-27 at 18 27 04" src="https://user-images.githubusercontent.com/189580/69749976-8defb780-1143-11ea-9368-00dd8d544391.png">

should be live soon™ :shipit:
